sdk/identity/azure-identity/tests/test_managed_identity.py

```diff
@@ -242,14 +242,14 @@ def test_azure_ml():
             Request(
                 url,
                 method="GET",
-                required_headers={"secret": secret, "User-Agent": USER_AGENT},
-                required_params={"api-version": "2017-09-01", "resource": scope},
+                required_headers={"X-IDENTITY-HEADER": secret, "User-Agent": USER_AGENT},
```

Comment:
Oops, I forgot about this! Azure ML imitates App Service 2017-09-01 in setting MSI_ENDPOINT and MSI_SECRET. Naturally it's a different implementation, and it returns `expires_on` as an integer. I suppose it's time to add e.g. `AzureMLCredential`.